tools/github_actions_install.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash -ef
+
+python setup.py build
+python setup.py install
```

Comment:
I wonder if we should have a job that checks that installation with pip runs smoothly?